lib/utils/Database.cpp

```diff
@@ -22,4 +31,18 @@ Database::~Database()
 {
 }
 
+bool
+Database::create(const std::string& file_name)
+{
+    std::cout << "Creating file: " << file_name << std::endl;
+}
+
+bool
+Database::open(
```

Comment:
Is there a particular reason why a concrete implementation for the `open()` and `create()` functions were added to the abstract Database class? 
If these are needed, then perhaps they should return `true` by default since I noticed a compiler warning about no return type:
```
/home/test/libROM/lib/utils/Database.cpp: In member function 'virtual bool CAROM::Database::create(const string&)':
/home/test/libROM/lib/utils/Database.cpp:38:1: warning: no return statement in function returning non-void [-Wreturn-type]
   38 | }
```
It might be good in the future to try and unify some code here such as checking if the file exists on `open()` / does not exist on `create()`, etc, if those logic checks are shared between HDF/CSV Database classes, but that is certainly work for another PR.

Reply:
Disregard my comment above. I just noticed those changes in your next PR #274.

Reply:
Yeah I didn't realize this was not implemented here but in PR #274. Thanks for catching that.